README.md

```diff
@@ -88,6 +88,44 @@ includes builds of this Go fork.
 * In Azure Linux 2.0, the package `msft-golang` installs this fork.
 * In Azure Linux 3.0, the `golang` package installs this fork.
 
+### Ubuntu
+
+To install the Microsoft Build of Go on Ubuntu, open a terminal and run the following commands:
+```bash
+# Valid values are only '22.04' and '24.04'
+# For other versions of Ubuntu, please use the tar.gz package
+ubuntu_release=`lsb_release -rs`
+wget https://packages.microsoft.com/config/ubuntu/${ubuntu_release}/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+sudo dpkg -i packages-microsoft-prod.deb
+```
+After the repository is added, install the Microsoft Build of Go by running the following commands:
+```bash
+sudo apt-get update
+sudo apt-get install msft-golang
+```
+Installing a Specific Version
+To install a specific version of the Microsoft Build of Go:
```

Comment:
Too much for the readme IMO. None of the other instructions in the readme file include how to get other versions.

(We could have a new file in `eng/docs` that we link to that goes into detail about installing specific Ubuntu packages if we really want to maintain this info.)